Javadoc errors cause build failure

Current master (ab0768c) cannot be built, due to (depends on JDK version used):

JDK 11:
```
.../src/main/java/jakarta/data/repository/Repository.java:31: error: reference not found
[ERROR]  * {@link jakarta.inject.Inject} annotation.<p>
.../src/main/java/jakarta/data/repository/Repository.java:37: error: malformed HTML
[ERROR]  * public interface Products extends DataRepository<Product, Long> {
.../src/main/java/jakarta/data/repository/Repository.java:37: error: bad use of '>'
[ERROR]  * public interface Products extends DataRepository<Product, Long> {
```

JDK 17 additionally:
```
.../src/main/java/jakarta/data/repository/Repository.java:68: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
.../src/main/java/jakarta/data/repository/Repository.java:259: error: attribute not supported in HTML5: valign
[ERROR]  * <tr valign="top"><td><code>existsBy...</code></td>
```

--- a/api/src/main/java/jakarta/data/repository/Repository.java
+++ b/api/src/main/java/jakarta/data/repository/Repository.java
@@ -28,13 +28,13 @@ import java.lang.annotation.Target;
  *
  * This class is a CDI bean-defining annotation when CDI is available,
  * enabling the container/runtime to the implementation available via the
- * {@link jakarta.inject.Inject} annotation.<p>
+ * <code>jakarta.inject.Inject</code> annotation.<p>
  *
- * For example,<p>
+ * For example,
  *
  * <pre>
  * &#64;Repository
- * public interface Products extends DataRepository<Product, Long> {
+ * public interface Products extends DataRepository&lt;Product, Long&gt; {
  *     List&lt;Product&gt; findByNameLike(String namePattern);
  *     ...
  * </pre>
@@ -63,9 +63,9 @@ import java.lang.annotation.Target;
  * You can also copy individual method signatures from the
  * built-in repository methods onto your own, which is possible
  * because the built-in repository methods are consistent with the
- * same set of conventions that you use to write custom repository methods.<p>
+ * same set of conventions that you use to write custom repository methods.
  *
- * <h3>Reserved Keywords for Name-Pattern-Based Repository Methods</h3>
+ * <h2>Reserved Keywords for Name-Pattern-Based Repository Methods</h2>
  *
  * TODO What follows will neither be correct nor complete in this initial issue/pull.
  * Its purpose is to figure out a structure into which we can document the
@@ -75,71 +75,71 @@ import java.lang.annotation.Target;
  * as a user. Some of that actual detail is very likely to be wrong, but
  * that's good because it will lead to the opening of issues for
  * clarifications and discussions to come up with a more comprehensive
- * and better documented solution.
+ * and better documented solution.<br><br>
  *
- * <table>
+ * <table style="width: 100%">
  * <caption><b>Reserved Method Name Prefixes</b></caption>
  * <tr>
- * <td valign="top"><b>Prefix</b></td>
- * <td valign="top"><b>Description</b></td>
- * <td valign="top"><b>Example</b></td>
+ * <td style="vertical-align: top"><b>Prefix</b></td>
+ * <td style="vertical-align: top"><b>Description</b></td>
+ * <td style="vertical-align: top"><b>Example</b></td>
  * </tr>
  *
- * <tr valign="top"><td><code>countBy</code> or just <code>count</code>?</td>
+ * <tr style="vertical-align: top"><td><code>countBy</code> or just <code>count</code>?</td>
  * <td>counts the number of entities</td>
  * <td><code>countByAgeGreaterThanEqual(ageLimit)</code></td></tr>
  *
- * <tr valign="top"><td><code>deleteBy</code></td>
+ * <tr style="vertical-align: top"><td><code>deleteBy</code></td>
  * <td>for delete operations</td>
  * <td><code>deleteByStatus("DISCONTINUED")</code></td></tr>
  *
- * <tr valign="top"><td><code>existsBy</code></td>
+ * <tr style="vertical-align: top"><td><code>existsBy</code></td>
  * <td>for determining existence</td>
  * <td><code>existsByYearHiredAndWageLessThan(2022, 60000)</code></td></tr>
  *
- * <tr valign="top"><td><code>findBy</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy</code></td>
  * <td>for find operations</td>
  * <td><code>findByHeightBetween(minHeight, maxHeight)</code></td></tr>
  *
- * <tr valign="top"><td><code>save</code></td>
+ * <tr style="vertical-align: top"><td><code>save</code></td>
  * <td>for save operations</td>
  * <td><code>save(Product newProduct)</code></td></tr>
  *
- * <tr valign="top"><td><code>updateBy</code></td>
+ * <tr style="vertical-align: top"><td><code>updateBy</code></td>
  * <td>TODO Is there a reason existing solutions don't seem to have this?</td>
  * <td><code>updateByIdSetModifiedOnAddPrice(productId, now, 10.0)</code></td></tr>
  * </table>
  * TODO When can "By" be omitted and "All" added? Need to document that.
- * <p>
+ * <br><br>
  *
- * <table>
+ * <table style="width: 100%">
  * <caption><b>Reserved Keywords</b></caption>
  * <tr>
- * <td valign="top"><b>Keyword</b></td>
- * <td valign="top"><b>Applies to</b></td>
- * <td valign="top"><b>Description</b></td>
- * <td valign="top"><b>Example</b></td>
+ * <td style="vertical-align: top"><b>Keyword</b></td>
+ * <td style="vertical-align: top"><b>Applies to</b></td>
+ * <td style="vertical-align: top"><b>Description</b></td>
+ * <td style="vertical-align: top"><b>Example</b></td>
  * </tr>
  *
- * <tr valign="top"><td><code>And</code></td>
+ * <tr style="vertical-align: top"><td><code>And</code></td>
  * <td>conditions</td>
  * <td>Requires both conditions to be satisfied in order to match an entity.
  * Precedence is determined by the data access provider.
  * TODO Or should we enforce a particular precedence?</td>
  * <td><code>findByNameLikeAndPriceLessThanEqual(namePattern, maxPrice)</code></td></tr>
  *
- * <tr valign="top"><td><code>Asc</code></td>
+ * <tr style="vertical-align: top"><td><code>Asc</code></td>
  * <td>sorting</td>
  * <td>Specifies ascending sort order for <code>findBy</code> queries</td>
  * <td><code>findByAgeOrderByFirstNameAsc(age)</code></td></tr>
  *
- * <tr valign="top"><td><code>Between</code></td>
+ * <tr style="vertical-align: top"><td><code>Between</code></td>
  * <td>numeric, strings, time</td>
  * <td>Requires that the entity's attribute value be within the range specified by two parameters.
  * The minimum is listed first, then the maximum.</td>
  * <td><code>findByAgeBetween(minAge, maxAge)</code></td></tr>
  *
- * <tr valign="top"><td><code>Contains</code></td>
+ * <tr style="vertical-align: top"><td><code>Contains</code></td>
  * <td>collections, strings</td>
  * <td> TODO Need to determine if I have a proper understanding of this one.
  * For Collection attributes, requires that the entity's attribute value,
@@ -149,61 +149,61 @@ import java.lang.annotation.Target;
  * <td><code>findByRecipientsContains(email)</code>
  * <br><code>findByDescriptionNotContains("refurbished")</code></td></tr>
  *
- * <tr valign="top"><td><code>Desc</code></td>
+ * <tr style="vertical-align: top"><td><code>Desc</code></td>
  * <td>sorting</td>
  * <td>Specifies descending sort order for <code>findBy</code> queries</td>
  * <td><code>findByAuthorLastNameOrderByYearPublishedDesc(surname)</code></td></tr>
  *
- * <tr valign="top"><td><code>EndsWith</code></td>
+ * <tr style="vertical-align: top"><td><code>EndsWith</code></td>
  * <td>strings</td>
  * <td>Requires that the characters at the end of the entity's attribute value
  * match the parameter value, which can be a pattern.</td>
  * <td><code>findByNameEndsWith(surname)</code></td></tr>
  *
- * <tr valign="top"><td><code>GreaterThan</code></td>
+ * <tr style="vertical-align: top"><td><code>GreaterThan</code></td>
  * <td>numeric, strings, time</td>
  * <td>Requires that the entity's attribute value be larger than the parameter value.</td>
  * <td><code>findByStartTimeGreaterThan(startedAfter)</code></td></tr>
  *
- * <tr valign="top"><td><code>GreaterThanEqual</code></td>
+ * <tr style="vertical-align: top"><td><code>GreaterThanEqual</code></td>
  * <td>numeric, strings, time</td>
  * <td>Requires that the entity's attribute value be at least as big as the parameter value.</td>
  * <td><code>findByAgeGreaterThanEqual(minimumAge)</code></td></tr>
  *
- * <tr valign="top"><td><code>In</code></td>
+ * <tr style="vertical-align: top"><td><code>In</code></td>
  * <td>all attribute types</td>
  * <td>Requires that the entity's attribute value be within the list that is the parameter value.</td>
  * <td><code>findByNameIn(names)</code></td></tr>
  *
- * <tr valign="top"><td><code>LessThan</code></td>
+ * <tr style="vertical-align: top"><td><code>LessThan</code></td>
  * <td>numeric, strings, time</td>
  * <td>Requires that the entity's attribute value be less than the parameter value.</td>
  * <td><code>findByStartTimeLessThan(startedBefore)</code></td></tr>
  *
- * <tr valign="top"><td><code>LessThanEqual</code></td>
+ * <tr style="vertical-align: top"><td><code>LessThanEqual</code></td>
  * <td>numeric, strings, time</td>
  * <td>Requires that the entity's attribute value be at least as small as the parameter value.</td>
  * <td><code>findByAgeLessThanEqual(maximumAge)</code></td></tr>
  *
- * <tr valign="top"><td><code>Like</code></td>
+ * <tr style="vertical-align: top"><td><code>Like</code></td>
  * <td>strings</td>
  * <td>Requires that the entity's attribute value match the parameter value, which can be a pattern.</td>
  * <td><code>findByNameLike(namePattern)</code></td></tr>
  *
- * <tr valign="top"><td><code>Not</code></td>
+ * <tr style="vertical-align: top"><td><code>Not</code></td>
  * <td>condition</td>
  * <td>Negates a condition.</td>
  * <td><code>deleteByNameNotLike(namePattern)</code>
  * <br><code>findByStatusNot("RUNNING")</code></td></tr>
  *
- * <tr valign="top"><td><code>Or</code></td>
+ * <tr style="vertical-align: top"><td><code>Or</code></td>
  * <td>conditions</td>
  * <td>Requires at least one of the two conditions to be satisfied in order to match an entity.
  * Precedence is determined by the data access provider.
  * TODO Or should we enforce a particular precedence?</td>
  * <td><code>findByPriceLessThanEqualOrDiscountGreaterThanEqual(maxPrice, minDiscount)</code></td></tr>
  *
- * <tr valign="top"><td><code>OrderBy</code></td>
+ * <tr style="vertical-align: top"><td><code>OrderBy</code></td>
  * <td>sorting</td>
  * <td>Sorts results of a <code>findBy</code> query according to one or more entity attributes.
  * Multiple attributes are delimited by <code>Asc</code> and <code>Desc</code>,
@@ -212,7 +212,7 @@ import java.lang.annotation.Target;
  * TODO could OrderBy be specified annotatively instead?</td>
  * <td><code>findByStatusOrderByYearHiredDescLastNameAsc(empStatus)</code></td></tr>
  *
- * <tr valign="top"><td><code>StartsWith</code></td>
+ * <tr style="vertical-align: top"><td><code>StartsWith</code></td>
  * <td>strings</td>
  * <td>Requires that the characters at the beginning of the entity's attribute value
  * match the parameter value, which can be a pattern.</td>
@@ -229,24 +229,24 @@ import java.lang.annotation.Target;
  * Hopefully it was at least varied enough to illustrate the sort of information to document.
  *
  * TODO The following reflects my own guessing about what return types could make sense.
- * Many are missed, including for reactive.<p>
+ * Many are missed, including for reactive.
  *
- * <table>
+ * <table style="width: 100%">
  * <caption><b>Return Types for Repository Methods</b></caption>
  * <tr>
- * <td valign="top"><b>Method</b></td>
- * <td valign="top"><b>Return Types</b></td>
- * <td valign="top"><b>Notes</b></td>
+ * <td style="vertical-align: top"><b>Method</b></td>
+ * <td style="vertical-align: top"><b>Return Types</b></td>
+ * <td style="vertical-align: top"><b>Notes</b></td>
  * </tr>
  *
- * <tr valign="top"><td><code>countBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>countBy...</code></td>
  * <td><code>long</code>, <code>Long</code>,
  * <br><code>int</code>, <code>Integer</code>,
  * <br><code>short</code>, <code>Short</code>,
  * <br><code>Number</code></td>
  * <td>Jakarta Persistence providers limit the maximum to <code>Integer.MAX_VALUE</code></td></tr>
  *
- * <tr valign="top"><td><code>deleteBy...</code>,
+ * <tr style="vertical-align: top"><td><code>deleteBy...</code>,
  * <br><code>updateBy...</code></td>
  * <td><code>void</code>, <code>Void</code>,
  * <br><code>boolean</code>, <code>Boolean</code>,
@@ -256,40 +256,40 @@ import java.lang.annotation.Target;
  * <br><code>Number</code></td>
  * <td>Jakarta Persistence providers limit the maximum to <code>Integer.MAX_VALUE</code></td></tr>
  *
- * <tr valign="top"><td><code>existsBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>existsBy...</code></td>
  * <td><code>boolean</code>, <code>Boolean</code></td>
  * <td></td></tr>
  *
- * <tr valign="top"><td><code>findBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy...</code></td>
  * <td><code>E</code>,
  * <br><code>Optional&lt;E&gt;</code></td>
  * <td>For queries returning a single item (or none)</td></tr>
  *
- * <tr valign="top"><td><code>findBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy...</code></td>
  * <td><code>E[]</code>,
  * <br><code>Iterable&lt;E&gt;</code>,
  * <br><code>Stream&lt;E&gt;</code>,
  * <br><code>Collection&lt;E&gt;</code></td>
  * <td></td></tr>
  *
- * <tr valign="top"><td><code>findBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy...</code></td>
  * <td><code>Collection</code> subtypes</td>
  * <td>The subtype must have a public default constructor and support <code>addAll</code> or <code>add</code></td></tr>
  *
- * <tr valign="top"><td><code>findBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy...</code></td>
  * <td><code>Page&lt;E&gt;</code>, <code>Iterator&lt;E&gt;</code></td>
  * <td>For use with pagination</td></tr>
  *
- * <tr valign="top"><td><code>findBy...</code></td>
+ * <tr style="vertical-align: top"><td><code>findBy...</code></td>
  * <td><code>LinkedHashMap&lt;K, E&gt;</code></td>
  * <td>Ordered map of Id attribute value to entity</td></tr>
  *
- * <tr valign="top"><td><code>save(E)</code></td>
+ * <tr style="vertical-align: top"><td><code>save(E)</code></td>
  * <td><code>E</code>,
  * <br><code>void</code>, <code>Void</code></td>
  * <td>For saving a single entity.</td></tr>
  *
- * <tr valign="top"><td><code>save(E...)</code>,
+ * <tr style="vertical-align: top"><td><code>save(E...)</code>,
  * <br><code>save(Iterable&lt;E&gt;)</code>,
  * <br><code>save(Stream&lt;E&gt;)</code></td>
  * <td><code>void</code>, <code>Void</code>,
@@ -302,9 +302,8 @@ import java.lang.annotation.Target;
  * <br>Collection subtypes must have a public default constructor
  * and support <code>addAll</code> or <code>add</code></td></tr>
  * </table>
- * <p>
  *
- * <h3>Parameters to Repository Methods</h3>
+ * <h2>Parameters to Repository Methods</h2>
  *
  * The parameters to a repository method correspond to the conditions that are
  * defined within the name of repository method (see reserved keywords above),
@@ -318,7 +317,7 @@ import java.lang.annotation.Target;
  * capabilities such as pagination and sorting.<p>
  *
  * TODO write this section after pagination and sorting and other aspects
- * are discussed and defined.<p>
+ * are discussed and defined.
  */
 @Documented
 @Retention(RetentionPolicy.RUNTIME)

